doc/source/overview_Rucio_account.rst

```diff
@@ -3,25 +3,18 @@ Rucio account
 -------------
 
 A Rucio account is the unit of assigning privileges in Rucio. It can
-represent individual users (like lgoossen, graemes, vgaronne, ...), a
-group of users (like bphys, higgs, susy, ...) or an organised
-production activity for the whole ATLAS collaboration (prod,
-tzero, ...). A Rucio account is identified by a string.
+represent individual users (such as user1, user2, user3, ...), a
+group of users (such as group1, group2, group3, ...) or a centralized production
+activity (such as datagen, wfms, ...). A Rucio account is identified by a string.
```

Comment:
wfms = the service account related to a workflow management system interfaced with Rucio

Reply:
Will add.